src/lineable.js

```diff
@@ -85,7 +85,7 @@
                 this.p.x = array[i][0]
                 this.p.y = array[i][1]
 
-                var p = this.p.matrixTransform(this.parent.node.getScreenCTM().inverse().multiply(this.el.node.getScreenCTM()));
+                var p = this.p.native().matrixTransform(this.parent.node.getScreenCTM().inverse().multiply(this.el.node.getScreenCTM()));
```

Comment:
just use `transform()` of SVG.Point here and pass it `this.parent.screenCTM()...` and multiply with `this.el.screenCTM()`.